src/installer/corehost/cli/corehost_context_contract.h

```diff
@@ -59,6 +61,7 @@ struct corehost_context_contract
     int (HOSTPOLICY_CALLTYPE *get_runtime_delegate)(
         coreclr_delegate_type type,
         /*out*/ void **delegate);
+    size_t unknown_delegate_type; // Added in 5.0
```

Comment:
If we return the first unknown I would call this `first_unknown_delegate_type`, or maybe `first_unsupported_delegate_type`.

Alternatively we could change the meaning to the last known/supported `last_supported_delegate_type`.

Reply:
I think I'm going to change the meaning and go with `last_supported_delegate_type`.